docs/griptape-cloud/structures/index.md

```diff
@@ -0,0 +1,44 @@
+# Structures
+
+Use [Structures](https://cloud.griptape.ai/structures) to host any python code in the cloud in seconds.
```

Comment:
> in seconds.

This is a big claim 😄

Reply:
*technically* it is seconds. Could be... 300ish seconds. Haha. But you make a good point in that this is technical documentation and not sales documentation. I can remove that part.